doc/source/install.rst

```diff
@@ -47,13 +47,43 @@ Execute the following command from the shell::
 
 Windows
 ```````
+Before installing the development version, uninstall the standard version of
```

Comment:
Okay. So if someone is using a virtual environment, they will have to do the same install procedure but after activating the virtual environment. Right? Should I add about that or just simply add the line you mentioned?